<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Credit</class>
 <widget class="QDialog" name="Credit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>431</width>
     <height>461</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab_1">
    <attribute name="title">
     <string>анн</string>
    </attribute>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Credit</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>%</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Deadline</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>100</y>
       <width>251</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>выполнить расчет</string>
     </property>
    </widget>
    <widget class="QLabel" name="repayment">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>210</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="overpayment">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>180</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="payment">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>150</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>150</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>monthly payment</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>180</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>loan overpayment</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>210</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>total repayment</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>10</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <double>100000000.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <double>100000000.000000000000000</double>
     </property>
    </widget>
    <widget class="QSpinBox" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>70</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>256</number>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>диф</string>
    </attribute>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>120</y>
       <width>241</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>выполнить расчет </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>0</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Credit</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>%</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Deadline</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>370</y>
       <width>150</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>400</y>
       <width>150</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>370</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>loan overpayment</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>400</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>total repayment</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>150</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>monthly payment</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>170</y>
       <width>221</width>
       <height>191</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>90</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>month start</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>0</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <double>100000000.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="lineEdit_5">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <double>100000000.000000000000000</double>
     </property>
    </widget>
    <widget class="QSpinBox" name="lineEdit_6">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>60</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>246</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="lineEdit_7">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>90</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>12</number>
     </property>
     <property name="singleStep">
      <number>0</number>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
